main/gdb/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gdb
-pkgver=8.2.1
-pkgrel=2
+pkgver=8.3
+pkgrel=0
 pkgdesc="The GNU Debugger"
 url="https://www.sourceware.org/gdb/"
 arch="all"
```

Comment:
empty variables can be removed for brevity

Reply:
GPL-3.0 is not a valid SPDX identifier

Reply:
Changed to `GPL-3.0-or-later LGPL-3.0-or-later`